<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>967</width>
    <height>721</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>IES</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_7">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabGeneral">
       <attribute name="title">
        <string>General</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QTextEdit" name="textEdit"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabMisc">
       <attribute name="title">
        <string>Misc</string>
       </attribute>
       <widget class="QGroupBox" name="groupBox">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>160</y>
          <width>651</width>
          <height>141</height>
         </rect>
        </property>
        <property name="title">
         <string>Info</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Number of</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="leNumberOf"/>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Input Watts</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QLineEdit" name="leWatts"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Lumen Per</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="leLumenPer"/>
         </item>
         <item row="1" column="2">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Ballast</string>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QLineEdit" name="leBallast"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Candela</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="leCandela"/>
         </item>
         <item row="2" column="2">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Futur Use/ Ballast lamp photometic</string>
           </property>
          </widget>
         </item>
         <item row="2" column="3">
          <widget class="QLineEdit" name="leFutur"/>
         </item>
        </layout>
       </widget>
       <widget class="QGroupBox" name="groupLuminous">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>10</y>
          <width>296</width>
          <height>132</height>
         </rect>
        </property>
        <property name="title">
         <string>Luminous dimensions</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="lblWidth">
           <property name="text">
            <string>Width:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="leWidth"/>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="lblWidthUnit">
           <property name="text">
            <string>Feet</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="lblLen">
           <property name="text">
            <string>Length:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="leLen"/>
         </item>
         <item row="1" column="2">
          <widget class="QLabel" name="lblLenUnit">
           <property name="text">
            <string>Feet</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="lblHeight">
           <property name="text">
            <string>Height:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="leHeight"/>
         </item>
         <item row="2" column="2">
          <widget class="QLabel" name="lblHeightUnit">
           <property name="text">
            <string>Feet</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QGroupBox" name="groupPhotometric">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>310</y>
          <width>901</width>
          <height>251</height>
         </rect>
        </property>
        <property name="title">
         <string>Photometric Type</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <widget class="QLabel" name="lblTypeA">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="lblTypeB">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="lblTypeC">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QGroupBox" name="groupUnits">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>10</y>
          <width>161</width>
          <height>141</height>
         </rect>
        </property>
        <property name="title">
         <string>Units</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <widget class="QRadioButton" name="rbFeet">
           <property name="text">
            <string>Feet</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QRadioButton" name="rbMeters">
           <property name="text">
            <string>Meters</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QLabel" name="lblImage">
        <property name="geometry">
         <rect>
          <x>510</x>
          <y>10</y>
          <width>221</width>
          <height>153</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tabSymmetry">
       <attribute name="title">
        <string>Symmetry</string>
       </attribute>
       <widget class="QGroupBox" name="groupHorizontal">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>40</y>
          <width>311</width>
          <height>441</height>
         </rect>
        </property>
        <property name="title">
         <string>Horizontal symmetry</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="0" column="0">
          <widget class="QRadioButton" name="rb0">
           <property name="text">
            <string>Laterally symmetric in all 
photometric planes.</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QRadioButton" name="rb1">
           <property name="text">
            <string>Symetric in each quadrant.</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QRadioButton" name="rb2">
           <property name="text">
            <string>Bilaterally symmetric about the 
0-180 deg photometric plane.</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QRadioButton" name="rb3">
           <property name="text">
            <string>Bilaterally symmetric about the 
90-270 deg photometric plane.</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QRadioButton" name="rb4">
           <property name="text">
            <string>No symmetry.</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QGroupBox" name="groupVertical">
        <property name="geometry">
         <rect>
          <x>460</x>
          <y>50</y>
          <width>251</width>
          <height>411</height>
         </rect>
        </property>
        <property name="title">
         <string>Vertical angles</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_6">
         <item row="0" column="0">
          <widget class="QRadioButton" name="rbV0">
           <property name="text">
            <string>The first angle is 0 deg 
 and the last is 90 deg.</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QRadioButton" name="rbV1">
           <property name="text">
            <string>The first angle is 0 deg 
 and the last is 180 deg.</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QRadioButton" name="rbV2">
           <property name="text">
            <string>The first angle is 90 deg 
 and the last is 180 deg.</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="tabData">
       <attribute name="title">
        <string>Data</string>
       </attribute>
       <widget class="QTableWidget" name="tableWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>60</y>
          <width>851</width>
          <height>511</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pbHplus">
        <property name="geometry">
         <rect>
          <x>710</x>
          <y>10</y>
          <width>38</width>
          <height>38</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QPushButton" name="pbHadd">
        <property name="geometry">
         <rect>
          <x>760</x>
          <y>10</y>
          <width>38</width>
          <height>38</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QPushButton" name="pbVadd">
        <property name="geometry">
         <rect>
          <x>870</x>
          <y>80</y>
          <width>38</width>
          <height>38</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QPushButton" name="pbVplus">
        <property name="geometry">
         <rect>
          <x>870</x>
          <y>130</y>
          <width>38</width>
          <height>38</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>967</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <widget class="QMenu" name="menuTheme">
     <property name="title">
      <string>Theme</string>
     </property>
     <addaction name="actionLight"/>
     <addaction name="actionDark"/>
    </widget>
    <addaction name="menuTheme"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionIsoLuxCurves"/>
    <addaction name="actionPolarCurves"/>
    <addaction name="action3DCurves"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionAdd_Vertical_Angle"/>
    <addaction name="actionAdd_Horizontal_Angle"/>
    <addaction name="actionDelete_Vertical_Angle"/>
    <addaction name="actionDelete_Horizontal_Angle"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuOptions"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="separator"/>
   <addaction name="actionOpen"/>
   <addaction name="separator"/>
   <addaction name="actionSave"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QToolBar" name="viewBar">
   <property name="windowTitle">
    <string>viewBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionIsoLuxCurves"/>
   <addaction name="separator"/>
   <addaction name="actionPolarCurves"/>
   <addaction name="separator"/>
   <addaction name="action3DCurves"/>
   <addaction name="separator"/>
  </widget>
  <action name="actionNew">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>icons/open.png</normaloff>icons/open.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionIsoLuxCurves">
   <property name="text">
    <string>IsoLux Curve</string>
   </property>
   <property name="toolTip">
    <string>IsoLuxCurves</string>
   </property>
  </action>
  <action name="actionPolarCurves">
   <property name="text">
    <string>Polar Curves</string>
   </property>
   <property name="toolTip">
    <string>PolarCurves</string>
   </property>
  </action>
  <action name="action3DCurves">
   <property name="text">
    <string>3D Curve</string>
   </property>
   <property name="toolTip">
    <string>3DCurves</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionLight">
   <property name="text">
    <string>Light</string>
   </property>
  </action>
  <action name="actionDark">
   <property name="text">
    <string>Dark</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAdd_Vertical_Angle">
   <property name="text">
    <string>Add Vertical Angle</string>
   </property>
  </action>
  <action name="actionAdd_Horizontal_Angle">
   <property name="text">
    <string>Add Horizontal Angle</string>
   </property>
  </action>
  <action name="actionDelete_Vertical_Angle">
   <property name="text">
    <string>Delete Vertical Angle</string>
   </property>
  </action>
  <action name="actionDelete_Horizontal_Angle">
   <property name="text">
    <string>Delete Horizontal Angle</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
